.gitmodules

```diff
@@ -25,3 +22,9 @@
 [submodule "src/extensions/default/JSLint/thirdparty/jslint"]
 	path = src/extensions/default/JSLint/thirdparty/jslint
 	url = https://github.com/peterflynn/JSLint.git
+[submodule "src/extensions/default/JavaScriptCodeHints/node/node_modules/acorn"]
```

Comment:
@swmitra 
we don't need submodules anymore, `npm install` will install dependencies defined in `src/extensions/default/JavaScriptCodeHints/package.json`, please remove